packages/selenium/test/RunPartial/FinishRunTests.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Linq;
+using System.IO;
+using Deque.AxeCore.Commons;
+using Deque.AxeCore.Selenium;
+using NUnit.Framework;
+using Newtonsoft.Json;
+
+namespace Deque.AxeCore.Selenium.Test.RunPartial
+{
+    public class FinishRunTests : TestBase
+    {
+        protected string axeForceLegacy { get; set; } = null;
+
+        [OneTimeSetUp]
+        public void SetUpForceLegacy()
+        {
+            axeForceLegacy = File.ReadAllText(
+                Path.Combine(TestContext.CurrentContext.TestDirectory, "fixtures/axe-force-legacy.js")
```

Comment:
NUnit doesn't technically guarantee that `TestDirectory` is equivalent to the output directory containing `fixtures/` - I think reusing `TestBase.FixturePath` would be preferable here both to avoid redundancy and because its directory traversal method is more reliable